hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -71,6 +71,9 @@
   @Parameter(names = {"--skip-ro-suffix"}, description = "Skip the `_ro` suffix for Read optimized table, when registering")
   public Boolean skipROSuffix = false;
 
+  @Parameter(names = {"--is-test-suite"}, description = "Hudi test suite")
```

Comment:
Let’s rename this to reflect what functionally it affects.. this does not look very nice

Reply:
Removed